client/src/assets/scss/base.scss

```diff
@@ -7,13 +7,25 @@
 /* @font-face should go here */
 
 /* HTML element styles should go here */
-
+.h6, h6 {
+  color: $dk-text;
+  font-size: 1rem;
+  line-height: 20px;
+  font-weight: 700;
+}
 /* global styles should go here */
-
-
 .pr-75 {
   padding-right: 75px !important;
 }
 .pl-75 {
   padding-left: 75px !important;
 }
+.action {
+  color: $error !important;
```

Comment:
Actually I'm not crazy about this classname, since the Existing Request Display displays a whole bunch of "actions" (buttons) that should not necessarily be red.
In a future commit, perhaps rename this to better denote what it's for?

Reply:
Absolutely, i'm not entirely sure of all the use cases but it appears in analyze results and existing request display templates. This the application of it.

<img width="616" alt="Screen Shot 2020-11-27 at 3 49 54 PM" src="https://user-images.githubusercontent.com/53542131/100489277-33779700-30c8-11eb-9911-0165dbd68516.png">

Reply:
100% tho, will update in future 👍